support/oidc-discovery-provider/README.md

```diff
@@ -75,37 +77,48 @@ will terminate if another domain is requested.
-|:--------------------------|-----------|-------------------------------------------------------------| ------- |
-| `named_pipe_name`| string | required  | Pipe name of the SPIRE Server API named pipe. Windows only. |         |
+| experimental      | Type   | Required? | Description                                                 | Default |
+| :---------------- | ------ | --------- | ----------------------------------------------------------- | ------- |
+| `named_pipe_name` | string | required  | Pipe name of the SPIRE Server API named pipe. Windows only. |         |
 
 #### Workload API Section
 
-| Key                | Type     | Required? | Description                               | Default |
-| ------------------ | -------- | --------- | ----------------------------------------- | ------- |
-| `experimental`     | section  | optional  | The experimental options that are subject to change or removal. | |
-| `socket_path`      | string   | required  | Path on disk to the Workload API Unix Domain socket. Unix platforms only. | |
-| `poll_interval`    | duration | optional  | How often to poll for changes to the public key material. | `"10s"` |
-| `trust_domain`     | string   | required  | Trust domain of the workload. This is used to pick the bundle out of the Workload API response. | |
+| Key             | Type     | Required? | Description                                                                                     | Default |
+| --------------- | -------- | --------- | ----------------------------------------------------------------------------------------------- | ------- |
+| `experimental`  | section  | optional  | The experimental options that are subject to change or removal.                                 |         |
+| `socket_path`   | string   | required  | Path on disk to the Workload API Unix Domain socket. Unix platforms only.                       |         |
+| `poll_interval` | duration | optional  | How often to poll for changes to the public key material.                                       | `"10s"` |
+| `trust_domain`  | string   | required  | Trust domain of the workload. This is used to pick the bundle out of the Workload API response. |         |
+
+| experimental      | Description                                             | Default |
+| :---------------- | ------------------------------------------------------- | ------- |
+| `named_pipe_name` | Pipe name of the Workload API named pipe. Windows only. |         |
+
+#### Health Checks Section
 
-| experimental                | Description                    | Default        |
-|:----------------------------|--------------------------------|----------------|
-| `named_pipe_name`           | Pipe name of the Workload API named pipe. Windows only. | |
+Health checks are enabled by adding `health_checks {}` to the configuration.
+The health checks endpoints runs at a dedicated listener on localhost.
+
```

Comment:
Maybe we can add some documentation here on how we determine liveness and readiness?

Reply:
I added documentation about how the states are determined. Please review it.

Reply:
Looks good, thanks!